package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harnessio/ff-nodejs-server-sdk",
-  "version": "1.2.15",
+  "version": "1.2.16",
```

Comment:
Looks like the lock file wasn't updated. Can you run `npm i` to update it and commit that?